getty/tools/mavenCalls.py

```diff
@@ -0,0 +1,44 @@
+import mvn, os
+
+def get_bin_path(hash):
+    os.sys_call("git checkout " + hash)
```

Comment:
can you add a "git_adapter" with a checkout(hash) method?

Reply:
done!